proposals/lip-0053.md

```diff
@@ -88,7 +87,7 @@ Setup to gather the required mainchain information:
 Create a cross-chain update transaction for a given height `h1`:
```

Comment:
Comment for the lines above:

- I don't think this section fits into the rationale, but could rather be an appendix or a non-protocol section in the specifications.
- For relayers running a mainchain node is not necessary, but only having access to the necessary endpoints.
- "Maintain a list of all CCMs included in the sidechain outbox. For each height where a CCM was included in the outbox, also save the inclusion witness of the outbox into the state root." : I don't think we need an inclusion proof for each message, but rather one for a group of messages. I think it would be better to specify more precisely which endpoint is needed here.
- "Maintain a history of all validator changes on the mainchain for rounds that have not yet been certified on the sidechain.": I think it would be better to specify more precisely which endpoint can be used here. Note that the BFT module stores the required information already.

Reply:
- As mentioned, I would prefer not to change this.
- Fixed.
- The inclusion witness is for the outbox root, not for the messages, I think the sentence is correct. 
- Wouldn't that be part of the validators module now? These endpoints are not defined yet (I think) so I would just keep this as it is.

Reply:
I moved it to an appendix.

Reply:
All good, I resolve this comment.